woocommerce/woocommerce-ios
# Upgrade to Xcode 15.4

## Description
Now that Xcode 15.4 is [supported](https://wp.me/paaHJt-6yv-p2) in CI, this PR makes changes to bump up the current Xcode version to 15.4.

## Steps to reproduce
N/A

## Testing information
Using Xcode 15.4:

- Check out the branch.
- run `rake dependencies`
- Clean build folder
- Run the UI tests against an iPad Air simulator

## Screenshots
N/A
